testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -782,7 +782,7 @@ private void assertTestingMaterializedViewQuery(String schema, String materializ
 
     private void createTestingMaterializedView(QualifiedObjectName view, Optional<String> comment)
     {
-        assertUpdate(format("CREATE SCHEMA IF NOT EXISTS %s", view.getSchemaName()));
+        assertUpdate(prepareCreateSchemaStatement(view.getSchemaName()));
```

Comment:
Does this belong to "Allow passing schema initializer to Iceberg query runner" commit?

(see other changes like this)

Reply:
Nope, moved one commit up.